packages/react-components/src/components/styles/MessageThread.styles.ts

```diff
@@ -107,8 +107,8 @@ export const defaultChatMessageContainer: ComponentSlotStyle = {
   minWidth: `${CHAT_MESSAGE_CONTAINER_MIN_WIDTH_REM}rem`,
   marginRight: '0rem',
   ' img': {
```

Comment:
(non-blocking for this PR) the `' img` and `' p'` here look to be causing these issues: [Bug 2777093](https://skype.visualstudio.com/SPOOL/_workitems/edit/2777093): [Web] logspam from fela is making it impossible to see any other logs

Is there a different css selector we can use that fela won't complain about?

Reply:
Thanks for reminding this! Will take some experiments and take that item if I find the reason after this PR